src/pages/academy/grading/subcomponents/GradingSubmissionsTable.tsx

```diff
@@ -147,8 +273,41 @@ const GradingSubmissionTable: React.FC<GradingSubmissionTableProps> = ({ submiss
     );
   }, [columnFilters, globalFilter, dispatch]);
 
+  // tells page to ask for new entries from main page when its state changes.
+  useEffect(() => {
+    updateEntries(limitGroup, pageParams, backendFilterParams);
+    // updateEntries seems to change between renders as a reference?
+    // eslint-disable-next-line
```

Comment:
DO NOT do this. There's a reason those warnings exist, when the `updateEntries` function is recreated (thus changes reference), you don't want to be calling the old reference, which may result in an error.